### PR TITLE
Get started btn

### PR DESCRIPTION
**Pull request by BladeSe7en**
Added an on hover effect to change the curser to a pointer when hovering over the get started button.
issue #103
Added href links in the navbar to /connect, /jobs, /shirts, /learn.
issue #102

### CODE REVIEW

**Comment by BladeSe7en** [comment]
Added a href link to https://www.sandiegojs.org/connect route from the pipeline project.

File: server/utils/getMeetups.js [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
-	const date =    `${currentDate.getFullYear()}-${currentDate.getMonth()}-${currentDate.getDay()}`;
+	const date = `${currentDate.getFullYear()}-${currentDate.getMonth()}-${currentDate.getDay()}`;
 		axios.get(`https://api.meetup.com/sandiegojs/events?no_later_than=${date}`)
-			.then(response => {
-			return	resolve(response.data.map(event => ({
-					meetupId: event.id,
-					name: event.name,
-					date: event.local_date,
-					time: event.local_time,
-					venue: event.venue,
```
Comment (elifnaz) [inline server/utils/getMeetups.js]: did you mean to delete venue here?

Comment (keizax) [reply server/utils/getMeetups.js]: @BladeSe7en was venue deleted intentionally?

**Comment by michaelerobertsjr** [comment]
Thank you for this contribution. It has been merged into the master branch.

**Pull request closed without merging**